samples/meetings-stage-view/nodejs/ClientApp/src/router/router.js

```diff
@@ -28,6 +29,7 @@ export const AppRoute = () => {
         <React.Fragment>
             <BrowserRouter>
                 <Routes>
+                    <Route path="/" element={<ShareToMeeting />}/>
                     <Route path="/appInMeeting" element={<AppInMeeting />}/>
                     <Route path="/configure" element={<Configure />}/>
                     <Route path="/home" element={<Home />}/>
```

Comment:
Looks like Spacing is not correct, please check once ?
`element={<ShareToMeeting />}`

`element= { <ShareToMeeting /> }`